src/libs/OptionsListUtils.ts

```diff
@@ -1812,13 +1815,18 @@ function getOptions(
 
     // Sorting the reports works like this:
     // - Order everything by the last message timestamp (descending)
+    // - When searching, self DM is put at the top
```

Comment:
We wanted to do this only when user have searched for themselves via their email or display name. But I think it is fine for now.